cmd/plugins_test.go

```diff
@@ -28,24 +31,59 @@ import (
 var initChan = make(chan struct{}, 256)
 var testDirRoot string
 
+const (
+	rootDir   string = "./"
```

Comment:
[nit] Do we _need_ the `string`?
```suggestion
	rootDir   = "./"
```

Reply:
Done